projects/FHC-AtmelStudio/Fussl-01/Fussl-01/doxySettings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DoxySettings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <SettingsFile>C:\Users\F-NET-ADMIN\Desktop\FHC-02\Fusselsoft-Home-Controller\projects\FHC-AtmelStudio\Fussl-01\Fussl-01\doxySettings.xml</SettingsFile>
+  <ProjectName>Fussl-01</ProjectName>
+  <DoxyExePath>C:\Program Files\doxygen\bin\doxygen.exe</DoxyExePath>
+  <UseDefaultSettings>true</UseDefaultSettings>
+  <GenerateAsPartOfBuild>true</GenerateAsPartOfBuild>
+  <ConfigFilePath>C:\Users\F-NET-ADMIN\Desktop\FHC-02\Fusselsoft-Home-Controller\projects\FHC-AtmelStudio\Fussl-01\Fussl-01\doxyfile</ConfigFilePath>
```

Comment:
This file contains user specific paths.
Check whether it will be created automatically. If so, add it to .gitignore.